cas-server-support-osf/src/main/java/org/pac4j/oauth/client/OrcidClient.java

```diff
@@ -16,39 +16,39 @@
 package org.pac4j.oauth.client;
 
 import org.apache.http.HttpStatus;
+
 import org.pac4j.core.context.WebContext;
 import org.pac4j.core.exception.HttpCommunicationException;
 import org.pac4j.oauth.client.exception.OAuthCredentialsException;
 import org.pac4j.oauth.profile.OAuthAttributesDefinitions;
-import org.pac4j.oauth.profile.XmlHelper;
 import org.pac4j.oauth.profile.orcid.OrcidAttributesDefinition;
 import org.pac4j.oauth.profile.orcid.OrcidProfile;
+import org.pac4j.oauth.profile.XmlHelper;
+
 import org.scribe.builder.api.OrcidApi20;
 import org.scribe.exceptions.OAuthException;
 import org.scribe.model.OAuthConfig;
+import org.scribe.model.ProxyOAuthRequest;
+import org.scribe.model.Response;
 import org.scribe.model.SignatureType;
 import org.scribe.model.Token;
-import org.scribe.oauth.ProxyOAuth20ServiceImpl;
 import org.scribe.tokens.OrcidToken;
-import org.scribe.model.ProxyOAuthRequest;
-import org.scribe.model.Response;
-
+import org.scribe.oauth.ProxyOAuth20ServiceImpl;
 
 /**
- * <p>This class is the OAuth client to authenticate users in ORCiD.</p>
- * <p>It returns a {@link org.pac4j.oauth.profile.orcid.OrcidProfile}.</p>
- * <p>More information at http://support.orcid.org/knowledgebase/articles/175079-tutorial-retrieve-data-from-an-orcid-record-with</p>
+ * The ORCiD Client.
+ *
+ * This class is the OAuth 2.0 client to authenticate users via ORCiD by authorization.
  *
  * @author Jens Tinglev
  * @author Michael Haselton
  * @author Longze Chen
- * @see org.pac4j.oauth.profile.orcid.OrcidProfile
  * @since 1.7.1
  */
 public class OrcidClient extends BaseOAuth20Client<OrcidProfile> {
 
     /** The default scope. */
-    protected static final String DEFAULT_SCOPE = "/orcid-profile/read-limited";
+    protected static final String DEFAULT_SCOPE = "/authenticate";
```

Comment:
Did orcid change the necessary scope, or did we get it wrong? A link to their scope docs might be helpful here

Reply:
This is just the default one provided by `pac4j` and it is no longer a valid scope. It will always be overwritten by the settings in `cas.properties` (which is `/authenticate`). I like the idea of adding the scope docs: https://members.orcid.org/api/oauth/orcid-scopes.